internal/pkg/util/fs/files/action_script.sh

```diff
@@ -68,9 +68,9 @@ restore_env() {
     # defined by docker or virtual file above, empty
     # variables are also unset
     for e in ${__exported_env__}; do
-        key=$(getenvkey "${e}")
+        key=${e%%=*}
         if ! test -v "${key}"; then
-            export "$(unescape ${e})"
+            export "${e%%=*}"
```

Comment:
should be `export "${e//'\n'/$IFS}"`

Reply:
Yes, thanks for pinpointing the problem.